.github/PULL_REQUEST_TEMPLATE/devconsole_pr.md

```diff
@@ -0,0 +1,29 @@
+
+# [JIRA-ID](add JIRA link): User Story / Defect / Task Title
+<!--For e.g [ODC-100](https://jira.coreos.com/browse/ODC-100): Title of user story / defect / task -->
+
+## Root cause
+<!--Briefly describe the root cause & analysis of the problem-->
+
+## Solution Description
+<!--Describe your code changes in detail and explain the solution-->
+
+- [ ] Is U/X approved?
+<!-- If designer review required, tag @openshift/team-devconsole-ux and add screenshots/gifs  -->
```

Comment:
Not sure of the value here for a checkbox. But every PR that affects the UI visuals should tag `@openshift/team-devconsole-ux` and include before (if useful) and after screenshots and gifs.

Reply:
Having a separate Screenshots/Gifs section would be helpful as most of our PRs do change UI some way or other.

Reply:
Agreed. Modified section as "Screen shots / Gifs for design review".